contracts/deployables/erc20/VotesERC20StakedV1.sol

```diff
@@ -133,6 +133,73 @@ contract VotesERC20StakedV1 is
         }
     }
 
+    function claimRewards(address _recipient) external virtual override {
+        for (uint256 i = 0; i < _rewardsTokens.length; ) {
+            _claimRewards(msg.sender, _recipient, _rewardsTokens[i]);
+
+            unchecked {
+                ++i;
+            }
+        }
+    }
+
+    function claimRewards(
+        address _recipient,
+        address[] memory _tokens
+    ) external virtual override {
+        for (uint256 i = 0; i < _tokens.length; ) {
+            if (!_rewardsTokenDatas[_tokens[i]].enabled)
+                revert InvalidRewardsToken(_tokens[i]);
+
+            _claimRewards(msg.sender, _recipient, _tokens[i]);
+
+            unchecked {
+                ++i;
+            }
+        }
```

Comment:
Would it save a bit of gas to create a local instance of the `_tokens[i]`, instead of pulling from the array twice?

```solidity
        for (uint256 i = 0; i < _tokens.length; ) {
            RewardsTokenData storage token = _tokens[i];
            if (!_rewardsTokenDatas[token].enabled)
                revert InvalidRewardsToken(_tokens[i]);

            _claimRewards(msg.sender, _recipient, token);

            unchecked {
                ++i;
            }
        }
```